### DISCUSSION TITLE
How to upgrade current project to new or latest version of ignite?

### DISCUSSION THREAD

**Discussion by chancesmith** [infinitered/ignite#1903] [Q&A] [answered]
We have a project built with Ignite three months ago. Now we want those sweet-hot changes that Ignite has to offers since we started. 😄🔥

Should we update Ignite?
What should we be concerned of?

**Comment by jamonholmgren** [comment, marked as answer]
@chancesmith We actually have a document for this!

https://github.com/infinitered/ignite/blob/master/docs/Upgrading.md

[permalink](https://github.com/infinitered/ignite/blob/b7e7a1fab0e476e826307e104a45ce4a207f680b/docs/Upgrading.md)

> Should we update Ignite?

I think so! The updates we're doing these days are much more around stability than any wholesale changes. With that said, some changes are on the menu for this year at some point, but we'll give plenty of warning and documentation around that.

> What should we be concerned of?

I don't think there's anything too big, to be honest. Like I said, mostly just stability and a few minor changes.

Thanks for your support @chancesmith !
